arviz/data/inference_data.py

```diff
@@ -252,6 +252,41 @@ def concat(*args, dim=None, copy=True, inplace=False, reset_dim=True):
     InferenceData
         A new InferenceData object by default.
         When `inplace==True` merge args to first arg and return `None`
+
+    Examples
+    --------
+    Use ``concat`` method to concatenate InferenceData objects. This will concatenates over
+    unique groups by default. We first create an InferenceData object:
+
+    .. ipython::
+
+        In [1]: import arviz as az
+           ...: import numpy as np
+           ...: data = {
+           ...:         "a": (["chain", "draw", "a_dim"], np.random.normal(size=(4, 100, 3))),
+           ...:         "b": (["chain", "draw"], np.random.normal(size=(4, 100))),
+           ...:         }
```

Comment:
We should follow black formatting, the `}` should be on the same level as the `d` in data and the two lines in between indented 4 spaces.